src/back/routes/notification.ts

```diff
@@ -63,36 +60,17 @@ async function sendNotification(req: WithAuth) {
     },
     timestamp: Date.now(),
   }))
-  try {
-    await DclNotificationService.sendNotifications(notifications)
-  } catch (error) {
-    ErrorService.report('Failed to send notification to DCL', {
-      notifications,
-      error: `${error}`,
-      category: ErrorCategory.Notifications,
-    })
-  }
-
-  try {
-    await NotificationService.sendNotification({
+  await Promise.all([
```

Comment:
What if this promise fails? Are we catching this somewhere?

Reply:
Each method in their service has a try-catch!